### DISCUSSION TITLE
fail target for up-poll

### DISCUSSION THREAD

**Discussion by sraturi** [unpoly/unpoly#343] [Q&A]
HelloI have a sample code 

```
<div class="status" up-poll up-source="/xyz/x/y/z/update" up-interval="5000">
  {#if status }
  <p class="status">{status}</p>
  {/if}
</div>
```

this works fine for polling. However, when i get a response with an error code, my status doesn't get updated, and the requests gets fired repeatedly with error : `No target selector given`

I tried setting up attributes like `up-fail-target` or `up-fail` but no luck at all. I am sending the same HTML page in both `success` and `failure`.  On success, I have no problem at all.

**Comment by sraturi** [comment]
Any suggestions on this?

**Comment by adam12** [comment]
Seems like it might be a bug.

Just to set up a proper repro, what is the specific error code that returns from the server?

**Reply by sraturi** [reply]
You can reproduce it with any error code, for my specific one, I had 409,415,408, etc.
